Source/JavaScriptCore/wasm/WasmIPIntGenerator.h

```diff
@@ -38,6 +39,168 @@ struct ModuleInformation;
 
 Expected<std::unique_ptr<FunctionIPIntMetadataGenerator>, String> parseAndCompileMetadata(std::span<const uint8_t>, const TypeDefinition&, ModuleInformation&, uint32_t functionIndex);
 
-} } // namespace JSC::Wasm
+} // namespace JSC::Wasm
+
+namespace IPInt {
+
+#pragma pack(1)
+
+// mINT: mini-interpreter / minimalist interpreter (whichever floats your boat)
+
+// Metadata structure for control flow instructions
+
+struct MDInstructionLength {
```

Comment:
`MD` might not be an abbreviation people reading the code are necessarily familiar with, maybe rename to `InstructionLengthMetadata` or a similar format (and ditto for the other structs)